### PR TITLE
Introduce end-of-line normalization

### PR DESCRIPTION
**Pull request by Pierre-Sassoulas**
## Description

This introduce end of line normalization. Some file were formatted for Windows.

## Type of Changes

|     | Type                   |
| --- | ---------------------- |
| ✓   | :bug: Bug fix          |
| ✓   | :hammer: Refactoring   |

## Related Issue

Required for #1441

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1 @@
+* text=auto
```
Comment (DanielNoord) [inline .gitattributes:1]: Shouldn't this have an empty new line? And perhaps a comment to explain what this does. Neat little function by the way.

Comment (Pierre-Sassoulas) [reply .gitattributes:2]: Regarding the end of line there's one, we have a hook to check that there is exactly one :)

Comment (DanielNoord) [reply .gitattributes:2]: Huh that's weird. I thought Github would display those in "file-view" or in "raw-view" but I didn't see one so I assumed it was missing.

Thanks for the comment!

File: tests/testdata/python3/data/recursion.py [hunk 2]
```diff
@@ -1,3 +1,3 @@
-""" For issue #25 """
-class Base(object):
+""" For issue #25 """
+class Base(object):
     pass
```
Comment (DanielNoord) [inline tests/testdata/python3/data/recursion.py:3]: There is a missing line missing here though @Pierre-Sassoulas 😄 

But no reason to add it now!

**Review by DanielNoord — APPROVED** [review]

**Pull request merged**